0000-ilp-ping-protocol/0000-ilp-ping-protocol.md

```diff
@@ -0,0 +1,42 @@
+# Interledger Ping Protocol
+
+## Introduction
+In order to monitor the health of an internetworked system, it is often useful to be able to send probe messages. In the Internet system, ICMP [echo](https://en.wikipedia.org/wiki/Ping_(networking_utility)#Echo_request) request and response packets are used together to verify connectivity (colloquially known as a "pinging" a host).  
+
+This RFC specifies a similar mechanism for Interledger nodes to test `uptime` and fulfill/reject `response-time` for a particular payment path, from the perspective of a payment sender.
+
+## Scope
+It is already possible to test unidirectional, end-to-end connectivity in Interledger by sending a small payment using existing mechanisms. However, doing so requires higher-level protocols, such as [SPSP](https://github.com/interledger/rfcs/blob/master/0009-simple-payment-setup-protocol/0009-simple-payment-setup-protocol.md) and [STREAM](https://github.com/interledger/rfcs/blob/master/0029-stream/0029-stream.md).
+
+Instead, this protocol uses only [Interledger-layer](https://github.com/interledger/rfcs/blob/master/0001-interledger-architecture/0001-interledger-architecture.md#interledger-protocol) technologies, plug a a known condition/fulfillment pair, thus allowing a sender to control the expected response type while at the same time preventing a sender from utilizing bandwidth without paying for it
```

Comment:
```suggestion
Instead, this protocol uses only [Interledger-layer](https://github.com/interledger/rfcs/blob/master/0001-interledger-architecture/0001-interledger-architecture.md#interledger-protocol) technologies, plug a a known condition/fulfillment pair, thus allowing a sender to control the expected response type while at the same time preventing a sender from utilizing bandwidth without paying for it.
```

Could you explain concretely the meaning of `preventing a sender from utilizing bandwidth without paying for it`?

Reply:
This part of the discussion is a bit out of date since the RFC has changed a bit. I'm going to resolve this discussion, but feel free to make a new comment if you still have questions.